integrated_channels/cornerstone/migrations/0025_alter_cornerstonelearnerdatatransmissionaudit_index_together.py

```diff
@@ -18,23 +17,44 @@ class Migration(migrations.Migration):
             ),
         ]
     else:
-        operations = [
-            migrations.SeparateDatabaseAndState(
-                state_operations=[
-                    migrations.AlterIndexTogether(
-                        name='cornerstonelearnerdatatransmissionaudit',
-                        index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
-                    ),
-                ],
-                database_operations=[
-                    migrations.RunSQL(sql="""
-                        CREATE INDEX cornerstone_cldta_85936b55_idx
-                        ON cornerstone_cornerstonelearnerdatatransmissionaudit (enterprise_customer_uuid, plugin_configuration_id)
-                        ALGORITHM=INPLACE LOCK=NONE
-                    """, reverse_sql="""
-                        DROP INDEX cornerstone_cldta_85936b55_idx
-                        ON cornerstone_cornerstonelearnerdatatransmissionaudit
-                    """),
-                ]
-            ),
-        ]
+        if 'postgresql' in db_engine:
+            operations = [
+                migrations.SeparateDatabaseAndState(
+                    state_operations=[
+                        migrations.AlterIndexTogether(
+                            name='cornerstonelearnerdatatransmissionaudit',
+                            index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+                        ),
+                    ],
+                    database_operations=[
+                        migrations.RunSQL(sql="""
+                            CREATE INDEX cornerstone_cldta_85936b55_idx
```

Comment:
Question: Why not `CREATE INDEX CONCURRENTLY` here?

Reply:
I believe  it will be fresh start of database if anyone will be choosing to go with Postgresql and there will be no pre-populated data. However If I use `CONCURRENTLY` there is a potential if could have error in background which is very unlikely but there is a chance. So I believe it will only add complexity with no benefits